ts/TextField/index.tsx

```diff
@@ -1,21 +1,54 @@
 import {Component, h} from 'preact';
 
-import Icon from '../Icon';
 import {MDCTextField} from '@material/textfield';
+import Icon from '../../Icon/index';
```

Comment:
shouldn't this be /ts/Icon?

Reply:
oh i see you haven't implemented that yet

Reply:
Once `Icon` is migrated, yes